git.go

```diff
@@ -50,6 +50,10 @@ func (g *GitClient) command(name string, arg ...string) *exec.Cmd {
 	cmd.Dir = g.Directory
 	cmd.Stdout = g.Output
 	cmd.Stderr = g.Output
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env,
+		"X_OAUTH_BASIC_TOKEN="+g.AccessToken,
+		"GIT_ASKPASS=/usr/local/bin/askpass.sh")
```

Comment:
Note this is hardcoded to the docker installation path, as the git documentation indicates this must be an absolute path.